src/mem/cache/tags/tagged_entry.hh

```diff
@@ -114,42 +101,28 @@ class TaggedEntry : public ReplaceableEntry
             isSecure(), isValid(), ReplaceableEntry::print());
     }
 
-  protected:
-    /**
-     * Set tag associated to this block.
-     *
-     * @param tag The tag value.
-     */
-    virtual void setTag(Addr tag) { _tag = tag; }
+    using CacheEntry::matchTag;
```

Comment:
Isn´t needing lines 105-108 a consequence of adding line 104? If you remove lines 104-108 won´t the base function be inaccessible?

Reply:
Bump

Reply:
I replied to your comment about why this is necessary. Let me know if it did not answer your question.

Reply:
Would it be possible to mark these as private then?

Reply:
Done. Although I am not sure how this works, since we have the same matchTag with different arguments. How do we mark one without secure tag private and make the one with is_secure tag to be public.